apps/alpine/metadata.yaml

```
@@ -0,0 +1,11 @@
+app: alpine
+base: true
+channels:
+- name: '3.18'
+  platforms:
+  - linux/amd64
+  - linux/arm64
+  stable: true
+  tests:
+    enabled: true
+    type: cli
```

```suggestion
  - name: '3.18'
    platforms:
      - linux/amd64
      - linux/arm64
    stable: true
    tests:
      enabled: true
      type: cli
```
It would be better for readability to have extra spaces here.